skops/card/_model_card.py

```diff
@@ -27,23 +31,44 @@ def _extract_estimator_config(model):
 
 
 def create_model_card(
-    model,
-    card_data,
+    path,
+    card_data=None,
     **card_kwargs,
 ):
     """Creates a model card for the model and saves it to the target directory.
 
     Parameters:
     ----------
-    model: estimator
-        scikit-learn compatible estimator.
-    card_data: CardData
+    path: str
```

Comment:
I think we should give people freedom to save their models themselves, I don't like this. we can add both path or model, it wouldn't be too bad.